beaupy/_beaupy.py

```diff
@@ -96,8 +98,8 @@ def prompt(
     """
     rendered = ''
     with _cursor_hidden(console), Live(rendered, console=console, auto_refresh=False, transient=True) as live:
-        value: List[str] = []
-        cursor_index = 0
+        value: List[str] = [initial_value] if initial_value else []
```

Comment:
There is a minor bug on this line, this needs to be:
```python
[*initial_value] if initial_value else []
```
rather than
```python
[initial_value] if initial_value else []
```

Otherwise the application raises `IndexError: list assignment index out of range` when I start a form with `initial_value='Hello, '` and press backspace.

Reply:
Great catch, thanks! I fixed the bug and added some more tests for edge cases.

Reply:
I thank you. I am going to merge this and do a release today.